### DISCUSSION TITLE
scaden can handle cell-subtype case

### DISCUSSION THREAD

**Discussion by Chuang1118** [KevinMenden/scaden#106] [Q&A] [answered]
Dear Author,

Thanks for this new API, especially has GPU option.

In #93, you mentioned "I never tried to estimate more than say 10-15 different celltypes ..."

In my opinion, high number of cell types doesn't matter if there is a low correlation between the cell types.
Now, I have a question:
Do you have any experience with a high correlation cell subtype within less than 10 difference celltypes ?
i.e. naive b cell, memory B, CD4+ and CD8+ T cells, Treg , Tfh in training data.

I'm looking foward to your reply.
Best,
Chuang

**Comment by KevinMenden** [comment, marked as answer]
Hi Chuang,

I guess you are right, it very much depends on how similar the cells are.
And to answer your question - no, I have not tried to deconvolve e.g. naive b cells from memory b cells and so on, as I had a different use case in mind for Scaden. So in my opinion, it will work, but the performance will degrade, similar as with other deconvolution methods. But unfortunately I cannot tell you howw good it will work.

(Sorry for the late reply by the way - holiday :) )

Cheers,
Kevin
